editor/plugins/node_3d_editor_plugin.h

```diff
@@ -794,7 +794,7 @@ class Node3DEditor : public VBoxContainer {
 
 	void set_can_preview(Camera3D *p_preview);
 
-	Node3DEditorViewport *get_editor_viewport(int p_idx) {
+	Node3DEditorViewport *get_editor_main_control(int p_idx) {
```

Comment:
It is actually referring to viewports here though, so the name was correct.

I'm not sure why this would trigger the crash reported in #44761 in CanvasItemEditorPlugin.